gr-filter/python/filter/design/filter_design.py

```diff
@@ -2166,7 +2165,7 @@ def action_open_dialog(self):
             self.gui.iirfilterTypeComboBox.setCurrentIndex(iirft[params["filttype"]])
             self.gui.iirfilterBandComboBox.setCurrentIndex(bandpos[params["bandtype"]])
             if params["filttype"] == "bessel":
-                critfreq = map(float, params["critfreq"][1:-1].split(','))
+                critfreq = list(map(float, params["critfreq"][1:-1].split(',')))
```

Comment:
My preference would be:
```suggestion
                critfreq = [float(x) for x in params["critfreq"][1:-1].split(',')]
```

...which I've seen quoted as being more Pythonic, and I think it is too. But this is personal preference, so keep whichever version you prefer.

Reply:
looks more elegant.